### PR TITLE
Fix deprecation

### PR DESCRIPTION
**Pull request by AlexKurek**

### CODE REVIEW

**Review by zacharyburnett — APPROVED** [review]
good catch!

File: src/stpipe/cli/command.py [hunk 1]
```diff
@@ -7,7 +7,8 @@ class Command(abc.ABC):
     be added to the _COMMAND_CLASSES list in core.py.
     """
 
-    @abc.abstractclassmethod
+    @abc.classmethod
```
Comment (zacharyburnett) [inline src/stpipe/cli/command.py:10]:
```suggestion
    @classmethod
```

File: src/stpipe/cli/command.py [hunk 2]
```diff
@@ -17,7 +18,8 @@ def get_name(cls):
         str
         """
 
-    @abc.abstractclassmethod
+    @abc.classmethod
```
Comment (zacharyburnett) [inline src/stpipe/cli/command.py:21]:
```suggestion
    @classmethod
```

File: src/stpipe/cli/command.py [hunk 3]
```diff
@@ -27,7 +29,8 @@ def add_subparser(cls, subparsers):
         subparsers : argparse._SubParsersAction
         """
 
-    @abc.abstractclassmethod
+    @abc.classmethod
```
Comment (zacharyburnett) [inline src/stpipe/cli/command.py:32]:
```suggestion
    @classmethod
```

Comment (AlexKurek) [reply src/stpipe/cli/command.py]: Done in https://github.com/spacetelescope/stpipe/commit/cef32eab5161f03c07dea93d67ce244ef69b7de3

**Comment by zacharyburnett** [comment]
sorry for the miscommunication! you should still include `@abc.abstractmethod`, like this:
```
@abc.abstractmethod
@classmethod
```

**Comment by AlexKurek** [comment]
> sorry for the miscommunication! you should still include `@abc.abstractmethod`, like this:
> 
> ```
> @abc.abstractmethod
> @classmethod
> ```

Ok, I get it now, done.

**Review by zacharyburnett — APPROVED** [review]

**Comment by zacharyburnett** [comment]
I apologize, I appear to have pushed away your fork's branch by accident. I remade this pull request at #167 with your commits

**Pull request closed without merging**